generatebundlefile/scripts/generate_bundle.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set -e
+set -x
+set -o pipefail
+
+export LANG=C.UTF-8
+
+IMAGE_REGISTRY="${1?Specify first argument - image registry}"
+KMS_KEY="${2?Specify second argument - kms key alias}"
+
+BASE_DIRECTORY=$(git rev-parse --show-toplevel)
+chmod +x ${BASE_DIRECTORY}/generatebundlefile/bin/generatebundlefile 
+
+# Faster way to install Cosign compared to go install github.com/sigstore/cosign/cmd/cosign@v1.5.1
+curl -s https://api.github.com/repos/sigstore/cosign/releases/latest \
+| grep 'browser_download_url.*cosign-linux-amd64"' \
+| cut -d '"' -f 4 \
+| tr -d \" \
+| wget -qi -
+
+chmod +x cosign-linux-amd64
+mv cosign-linux-amd64 /usr/bin/cosign
+cosign version
+
+# Create the bundle
+${BASE_DIRECTORY}/generatebundlefile/bin/generatebundlefile  \
+    --input ${BASE_DIRECTORY}/generatebundlefile/data/input_120.yaml
+
+# Sign the Bundle
+export AWS_REGION="us-west-2"
+SIGNATURE=$(cosign sign-blob --key awskms:///alias/${KMS_KEY} output/bundle-1.20.yaml)
+
+# Add signature annotation
+${BASE_DIRECTORY}/generatebundlefile/bin/generatebundlefile  \
+    --input ${BASE_DIRECTORY}/generatebundlefile/output/bundle-1.20.yaml \
+    --signature ${SIGNATURE}
+
+# Oras Download
+curl -LO https://github.com/oras-project/oras/releases/download/v0.12.0/oras_0.12.0_linux_amd64.tar.gz
+mkdir -p oras-install/
+tar -zxf oras_0.12.0_*.tar.gz -C oras-install/
+mv oras-install/oras /usr/local/bin/
+rm -rf oras_0.12.0_*.tar.gz oras-install/
+
+# Push Oras Bundle
+ECR_PASSWORD=$(aws ecr-public get-login-password --region us-east-1 | tr -d '\n')
+# Go into same directory as the file when performing ORAS push otherwise it expands the pull with full directory structure
+cd output/
```

Comment:
Probably no user/pass here if we can help it and hard coded bundle name

Reply:
I'll look into ORAS and if there is a cleaner way here.

Reply:
So it works without using the password input if you have ecr-credential helper installed, but it seems like installing another piece of software is not a good idea here. 
https://github.com/awslabs/amazon-ecr-credential-helper/issues/102

```

oras login --username AWS --password-stdin public.ecr.aws
--
225 | + aws ecr-public get-login-password --region us-east-1
226 | Error: error storing credentials - err: exit status 1, out: `not implemented`
```